domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/EvidenceItem.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+import java.util.Objects;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class EvidenceItem {
+    private final EvidenceType type;
+    private final String description;
+
+    public EvidenceItem(EvidenceType type, String description) {
+        this.type = type;
+        this.description = description;
+    }
+
+    public EvidenceType getType() {
+        return type;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (this == other) {
+            return true;
+        }
+        if (other == null || getClass() != other.getClass()) {
+            return false;
+        }
+        EvidenceItem evidenceItem = (EvidenceItem) other;
```

Comment:
Small thing but usually we call variable `that`, would be great to update for consistency.